usaspending_api/etl/management/commands/load_submission.py

```diff
@@ -139,7 +139,10 @@ def handle(self, *args, **options):
         # Once all the files have been processed, run any global
         # cleanup/post-load tasks.
         # 1. Load subawards
-        load_subawards(submission_attributes, db_cursor)
+        try:
```

Comment:
The reason I wrapped this portion in it's own block is because it's a totally separate process from the rest of the data load - if it errors out, there's no reason not to continue on and update the award transactions/etc. using the other helpers. 

We can also just remove it from load_submissions all together, and then just call load_subawards separately.